warn/scrapers/mi.py

```diff
@@ -1,6 +1,7 @@
 import re
```

Comment:
The Michigan scraper shouldn't be in the HI PR. Can we pull this out?

Reply:
Sure!

Reply:
The Michigan scraper is now being deleted. Isn't it already in the main branch? If so, we don't want to remove it here. Can you get it back in, unedited?